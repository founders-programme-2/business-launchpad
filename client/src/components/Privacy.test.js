import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';

const Privacy = () => {
  return <h1>Hello</h1>;
};

export default Privacy;

test('Natural', () => {
  const privacy = renderer
    .create(
      <BrowserRouter>
        {' '}
        <privacy />{' '}
      </BrowserRouter>
    )
    .toJSON();
  expect(privacy).toMatchSnapshot();
});
