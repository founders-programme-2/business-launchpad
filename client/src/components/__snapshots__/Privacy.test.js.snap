// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  " ",
  <privacy />,
  " ",
]
`;
